ios/Mattermost/Info.plist

```diff
@@ -64,25 +64,25 @@
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Share post data across devices with Mattermost</string>
 	<key>NSCalendarsUsageDescription</key>
-	<string>Share contacts in your Mattermost instance</string>
+	<string>Share contacts in your Mattermost workspace</string>
 	<key>NSCameraUsageDescription</key>
-	<string>Take a Photo or Video and upload it to your Mattermost instance</string>
+	<string>Take a Photo or Video and upload it to your Mattermost workspace</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Mattermost uses Face ID to restrict unauthorized users from accessing the app.</string>
 	<key>NSLocationAlwaysUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
```

Comment:
```suggestion
	<string>Send your current location to Mattermost</string>
```

Reply:
Question: Does a mobile user **send** their current location to an app, or do they **share** their current location with an app?

Reply:
This happens within the Mattermost app itself.  No external app is involved.  The code would request the permission to the user to allow it to get its location.  For example, it allows the MM app to know which country the user is in.

Reply:
Thanks, @avinashlng1080! Given the context you provided, I'm leaning towards "share" rather than "send".